### PR TITLE
caja: add page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** latest

### CODE REVIEW

File: pages/linux/caja.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# caja
+
+> Manages files and directories in MATE desktop environment.
+> More information: <https://manned.org/caja.1>.
```
Comment (navarroaxel) [inline pages/linux/caja.md:4]:
```suggestion
> More information: <https://manned.org/caja>.
```

File: pages/linux/caja.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Open a current user home directory:
```
Comment (navarroaxel) [inline pages/linux/caja.md:6]:
```suggestion
- Open and display the home directory for the current user:
```

File: pages/linux/caja.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`caja`
+
+- Open specific directories in separate windows:
+
+`caja {{path/to/directory{1,2,...}}}`
```
Comment (navarroaxel) [inline pages/linux/caja.md:12]: Mixing this example with specific bash syntax is hard to understand for new users
```suggestion
`caja {{path/to/directory1 path/to/directory12 ...}}}`
```

Comment (EmilySeville7cfg) [reply pages/linux/caja.md:12]: I don't understand. In some pages it is required to use brace expansion and in others - not. We have to describe a formal rule when to use it.

Comment (EmilySeville7cfg) [reply pages/linux/caja.md:12]: #7969 - please consider this issue

File: pages/linux/caja.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages/linux/caja.md:6]: I've used just: `Open the current user home directory`.

File: pages/linux/caja.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# caja
+
+> Manages files and directories in MATE desktop environment.
+> More information: <https://manned.org/caja>.
+
+- Open the current user home directory:
+
+`caja`
+
+- Open specific directories in separate windows:
```
Comment (navarroaxel) [inline pages/linux/caja.md:10]:
```suggestion
- Open specific directories in a separate windows:
```

Comment (EmilySeville7cfg) [reply pages/linux/caja.md:10]: But `windows` is a plural noun. 😄

Comment (navarroaxel) [reply pages/linux/caja.md:10]: oh, ok.

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**